dashboard/app/dsl/base_dsl.rb

```diff
@@ -39,4 +39,22 @@ def i18n_hash
   def i18n_strings
   end
 
+  def self.boolean(name)
+    define_method(name) do |val|
+      instance_variable_set "@#{name}", ActiveRecord::ConnectionAdapters::Column::value_to_boolean(val)
+  	end
+  end
+
+  def self.string(name)
+    define_method(name) do |val|
+      instance_variable_set  "@#{name}", val
+  	end
+  end
+
+  def self.integer(name)
+    define_method(name) do |val|
+      instance_variable_set "@#{name}", ActiveRecord::ConnectionAdapters::Column::value_to_integer(val)
+  	end
```

Comment:
Lint/BlockAlignment: `end` at 57, 3 is not aligned with `define_method(name) do |val|` at 55, 4<br>Style/Tab: Tab detected.